### PR TITLE
Update SGX and FPGA webhook flags

### PR DESCRIPTION
**Pull request by mythi**
Fixes: #1064

### CODE REVIEW

**Review by hj-johannes-lee — APPROVED** [review]
LGTM!

File: deployments/fpga_admissionwebhook/rbac/leader_election_role.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# permissions to do leader election.
+apiVersion: rbac.authorization.k8s.io/v1
```
Comment (tkatila) [inline deployments/fpga_admissionwebhook/rbac/leader_election_role.yaml:2]: I'm curious, did the leader election work previously?

**Review by tkatila — APPROVED** [review]
lgtm

**Review by bart0sh — APPROVED** [review]
/lgtm

File: deployments/fpga_admissionwebhook/rbac/leader_election_role.yaml (hunk 1 above)
Comment (mythi) [reply deployments/fpga_admissionwebhook/rbac/leader_election_role.yaml:2]: looks like no and it's still broken. I have a fix available soon.

**Pull request merged**